api/python/cellxgene_census/tests/test_open.py

```diff
@@ -38,15 +48,21 @@ def test_open_soma_stable() -> None:
 
 
 @pytest.mark.live_corpus
-def test_open_soma_latest() -> None:
+def test_open_soma_latest(requests_mock: rm.Mocker) -> None:
+    requests_mock.real_http = True
+    requests_mock.get(CELL_CENSUS_MIRRORS_DIRECTORY_URL, json=mirrors_json)
+
     # There should _always_ be a 'latest'
     with cellxgene_census.open_soma(census_version="latest") as census:
         assert census is not None
         assert isinstance(census, soma.Collection)
 
 
 @pytest.mark.live_corpus
-def test_open_soma_with_context() -> None:
+def test_open_soma_with_context(requests_mock: rm.Mocker) -> None:
```

Comment:
this is using the old (deprecated) interface (e.g., `s3_region`).  We should test the new interface (or both)

Reply:
Since this is a `live_corpus` test, we can update this after the changes to release.json, otherwise it will fail. Agree we should switch to the new interface though.